### ISSUE TITLE
Signature Issue Or Flow might harm your device

### ISSUE THREAD

**Issue by Kalaschnik** [Flow-Launcher/Flow.Launcher#117]
After downloading Flow I always get:
![image](https://user-images.githubusercontent.com/9831678/86727646-e701c100-c02b-11ea-805e-afb7459dbd7a.png)

In the downloads page of my browser (Edge) I need to confirm (show more) that this app is really safe:
![image](https://user-images.githubusercontent.com/9831678/86727877-1fa19a80-c02c-11ea-9c4d-d0c9b1e3601d.png)

This is probably some certificate stuff? Do you have any experience how to "solve" this behavior?

**Comment by Kalaschnik** [comment]
This might be Microsoft Edge related to their SmartScreen Filter. You can click on Report this app as safe and enter some information:
![image](https://user-images.githubusercontent.com/9831678/86772025-84bcb680-c053-11ea-8082-f79c5cf7a2cf.png)

Something @jjw24 you want to consider to do?

**Comment by jjw24** [comment]
Yep will do. I haven't sorted out this issue before so any tips is welcome

**Comment by Kalaschnik** [comment]
rel:
https://stackoverflow.com/questions/48946680/how-to-avoid-the-windows-defender-smartscreen-prevented-an-unrecognized-app-fro

**Comment by jjw24** [comment]
From Win10 after download and run installer:
![image](https://user-images.githubusercontent.com/26427004/90473097-6f17d200-e165-11ea-8cd0-5da4bd34763d.png)

**Comment by Kalaschnik** [comment]
I just red:

> If you have a standard code signing certificate, some time will be needed for your application to build trust. Microsoft affirms that an Extended Validation (EV) Code Signing Certificate allows us to skip this period of trust-building. According to Microsoft, extended validation certificates allow the developer to immediately establish a reputation with SmartScreen. Otherwise, the users will see a warning like "Windows Defender SmartScreen prevented an unrecognized app from starting. Running this app might put your PC at risk.", with the two buttons: "Run anyway" and "Don't run".
https://stackoverflow.com/a/51113771/2258480

Basically (without paying $400 or something), SmartScreen just need some time to see that this app was used by a number of people to built trust, if I understand it correctly.

**Comment by jjw24** [comment]
Yeah that's what my understanding was at previously when I was researching on this. Will take another look sometime later. But if we have to pay that fee, probably not ideal.

**Comment by barryseymour** [comment]
Still getting this in November 2024. My work doesn't allow the 'Run Anyway' option so I'm SOL until this is resolved.
